lib/models/room.js

```diff
@@ -627,14 +667,18 @@ function setEventMetadata(event, stateContext, toStartOfTimeline) {
  * @param {Room} room The matrix room.
  * @param {string} userId The client's user ID. Used to filter room members
  * correctly.
+ * @param {bool} implicit Return the implicit room name that we'd see if there
+ * was no m.room.name event.
```

Comment:
Why is this called `implicit`? All this flag does is ignore the `m.room.name` event if it exists, so can we perhaps call it something more descriptive like `ignoreRoomNameEvent`?